jest.config.js

```diff
@@ -0,0 +1,26 @@
+const config = {
+  'moduleNameMapper': {
+    // Maps all media files to a mock file.
+    // Without this, test will crash with a "Jest encountered an unexpected token" due to Jest having no idea what to do with binary files.
+    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': '<rootDir>/jest/mockMediaFile.js',
+
+    // Maps the @src alias added in webpack.
+    '^@src/(.*)$': '<rootDir>/src/$1'
+  },
+
+  // Tells Jest to look for unit test files in the /src directory.
+  'roots': [
+    '<rootDir>/src'
+  ],
+
+  'setupFilesAfterEnv': [
+    '<rootDir>/jest/jest-setup.js'
+  ],
+
+  // Without this, render() will result in "document is not defined" errors.
+  'testEnvironment': 'jsdom',
+}
+
+// Jest doesn't like 'export' (i.e. ES6 modules), so we use module.exports (CommonJS modules).
+// Probably because it's running in Node.js.
```

Comment:
Node supports `export` but only for ESM, ie. `type: 'module'` in `package.json`.

Reply:
This repo's `package.json` doesn't declare the package type, so Node and Jest default to CommonJS.

Reply:
Thanks Jim, that's very helpful to know! I didn't realise Node.js could support ESM.

I took some time to explore converting this package to an ESM, [but that's gonna be a work in progress.](https://github.com/zooniverse/community-catalog/issues/165)

Reply:
Pure ESM might be a good idea when starting a new project, but it's definitely a fair bit of work converting an existing package.

Reply:
Also, #153 is failing the tests because `@zooniverse/react-components` now depends on d3, for SVG data subjects, and d3 is pure ESM.